reconcile/openshift_groups.py

```diff
@@ -44,6 +44,9 @@ def get_cluster_state(group_items, oc_map):
     results = []
     cluster = group_items["cluster"]
     oc = oc_map.get(cluster)
+    if not oc:
+        logging.log(level=oc.log_level, msg=oc.message)
+        return None
```

Comment:
this will break ahead, because the `fetch_current_state()` loops through the return from `get_cluster_state()` , assuming it's a list (line 94 below).